ui/src/components/AppHeader.js

```diff
@@ -1,56 +1,69 @@
 import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 
-import AppBar from '@material-ui/core/AppBar';
 import IconButton from '@material-ui/core/IconButton';
-import Toolbar from '@material-ui/core/Toolbar';
-import Typography from '@material-ui/core/Typography';
 import MenuIcon from '@material-ui/icons/Menu';
 
 const useStyles = makeStyles(theme => ({
-  appBar: ({ drawerWidth }) => ({
+  appBar: {
     position: 'sticky',
-    ...(drawerWidth && {
-      [theme.breakpoints.up('sm')]: {
-        width: `calc(100% - ${drawerWidth}px)`,
-        marginLeft: drawerWidth,
-      },
-    }),
-  }),
+    [theme.breakpoints.up('sm')]: {
+      display: 'none',
+    },
+    padding: '8px',
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
   menuButton: ({ drawerWidth }) => ({
     marginRight: theme.spacing(2),
     ...(drawerWidth && {
       [theme.breakpoints.up('sm')]: {
         display: 'none',
       },
     }),
+    height: '48px',
   }),
-  title: {
-    flexGrow: 1,
+  logo: {
+    height: '60px',
+    width: '200px',
+    display: 'flex',
+    alignItems: 'center',
+    '& a': {
```

Comment:
what is this `'& a'` syntax?

Reply:
This is a nested selector. It's basically "select any anchor tag within the selector I'm already in". It works similar to sass https://css-tricks.com/the-sass-ampersand/